src/docs/schema_element_deprecation_guide.md

```diff
@@ -0,0 +1,26 @@
+# Deprecating NMDC schema elements
+
+When deprecating schema elements, it is a two step process:
+
+1. For one release cycle, the schema will include deprecation annotations to the element's definition and commenting out any usages of the slot (e.g. in a class, domain, range, etc) including a deprecation date (e.g. `# doi_awards deprecated on 11/12/2023` in a class's usage)
```

Comment:
let's encourage our teammates to use iso8601/international dates like 2023-11-12

Reply:
I added this.